Improve text-overflow in layers panel

ezxyz opened the pull request:
Adding ellipsis to the text layers.

Fixes #1066

Commit:
Update layer.js

Adding ellipsis to the text layers.

swissspidy commented on assets/src/edit-story/components/panels/layer/layer.js:
```diff
@@ -121,8 +124,8 @@ function Layer({ layer }) {
-          <LayerIcon />
-        )}
+            <LayerIcon />
+          )}
```
The linter complains about those two changed lines here.

ezxyz replied:
Fixed, I accidentally used a different formatter than Prettier.

Commit:
Update layer.js

Fixed an accidental auto-format due to me not using Prettier formatter by default.
